src/Network/Freddy.hs

```diff
@@ -0,0 +1,48 @@
+{-# OPTIONS -XOverloadedStrings #-}
+module Network.Freddy (connect, respondTo) where
+
+import Network.AMQP
+import Data.Text (Text, pack)
+import Data.ByteString.Lazy.Char8 (ByteString)
+import Network.Freddy.Result
+
+connect :: String -> Text -> Text -> Text -> IO Connection
+connect host vhost user password =
+  openConnection host vhost user password
```

Comment:
I might be speaking from ignorance, but after reading [this thing](http://www.stephendiehl.com/posts/production.html) this morning I think at least this line and all other AMQP identifiers should be using a qualified import.